### DISCUSSION TITLE
New install with Nvidia GPU

### DISCUSSION THREAD

**Discussion by wifispray** [blakeblackshear/frigate#15196] [Ask A Question]
Hey Everyone.

I have a new PC I wanted to use for running frigate (with a bunch of other stuff).

It has an NVIDIA 3060 GPU, its the first time using this with Frigate. Does anyone have a docker-compose and config file (scrubbed) they might be able to share to help get me started. I believe this will be a slightly different setup from my current Intel based setup.

Few questions I have, should I use the tensorrt or the jp4 or jp5?
Can I also still use the Coral device I have or would this be pointless with a powerful GPU?
Any Gotcha would be great :)

**Comment by hawkeye217** [comment]
The docs have you covered: https://docs.frigate.video/configuration/object_detectors#nvidia-tensorrt-detector

The setup is much more complex than a Coral. If your Coral has been working well, there's probably no reason to stop using it. It's much less power hungry and just as fast for object detection.

**Comment by wifispray** [comment]
Something like this @hawkeye217 

services:
  frigate:
    image: "ghcr.io/blakeblackshear/frigate:0.15.0-beta2-tensorrt"
    container_name: "frigate"
    domainname: "yours.com"
    hostname: "frigate"
    privileged: true
    restart: unless-stopped
    shm_size: "512mb"
    devices:
    volumes:
      - "/etc/localtime:/etc/localtime:ro"
      - "/home/docker/Documents/frigate/config:/config"
      - "/home/docker/Documents/frigate/media:/media/frigate"
      - type: tmpfs
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    network_mode: "host"
    environment:
      - TRT_MODEL_PREP_DEVICE=0
      - YOLO_MODELS=yolov7-320
      - USE_FP16=false
